test/e2e/metering_manual_install_tests.go

```diff
@@ -181,3 +181,34 @@ func s3InstallFunc(ctx *deployframework.DeployerCtx) error {
 
 	return nil
 }
+
+// customNodeSelectorFunc is a test helper function that adds a
+// custom testing label to all of the worker nodes in a particular
+// cluster. This is intended to be run prior to firing off any
+// Metering installations, or running any post-install tests,
+// that require testing this kind of configuration.
+func customNodeSelectorFunc(ctx *deployframework.DeployerCtx) error {
+	nodes, err := ctx.Client.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{
+		LabelSelector: "node-role.kubernetes.io/worker=",
+	})
+	if err != nil || len(nodes.Items) == 0 {
+		return fmt.Errorf("Failed to list the worker nodes in the cluster: %v", err)
+	}
+
+	nodeTestingLabelKey := "metering-node-testing-label"
+	for _, node := range nodes.Items {
```

Comment:
I wasn't sure if there was a better or cleaner way of doing this. Essentially this marks every worker node with the testing node label, which should be a safe operation and we attempt to remove this label after we spin down the e2e suite.